snapcraft/plugins/_python/_pip.py

```diff
@@ -196,7 +197,8 @@ def _is_pip_installed(self):
             # Using _run_output here so stdout doesn't get printed to the
             # terminal.
             self._run_output([], stderr=subprocess.STDOUT)
-        except subprocess.CalledProcessError as e:
+        except (subprocess.CalledProcessError,
+                SnapcraftPluginCommandError) as e:
```

Comment:
Note that `self._run{_output}` here actually goes through `common.run{_output}` as it's not part of a plugin. You shouldn't need to be handling `SnapcraftPluginCommandErrors` here.